### PR TITLE
Add GLPI 9.5 compatibility

### PR DESCRIPTION
**Pull request by cedric-anne**

### CODE REVIEW

**Comment by sworlls** [comment]
Is this possible to add tag to be used with notification on this new version?

File: tests/FieldsDbTestCase.php [hunk 1]
```diff
@@ -34,26 +34,15 @@
...
-         foreach ($files as $fileinfo) {
-            $todo = ($fileinfo->isDir() ? 'rmdir' : 'unlink');
-            $todo($fileinfo->getRealPath());
-         }
       }
+      parent::beforeTestMethod($method);
+   }
 
-      parent::setUp();
+   public function afterTestMethod($method) {
```
Comment (trasher) [inline tests/FieldsDbTestCase.php]: Is this one called when tests fails? As far as I remember, I've setted up files removal in `setUp` to be sure files was all removed on each run.

Comment (cedric-anne) [reply tests/FieldsDbTestCase.php]: I reverted this part.

**Comment by cedric-anne** [comment]
Included in #373

**Pull request closed without merging**